sft-n-hrd.js

```diff
@@ -1,28 +1,27 @@
-// //function addNumbers(x, y){
-// return x + y
-// }
+// Event listener para las cajas de input
+const enterButton= document.getElementById("enterbutton");
 
-// const totalAge = addNumbers(30, 31);
-
-// console.log(totalAge);
-var inchesToCmsConverstionFactor = 2.54
-const brianLenghtInches = 4.5;
-const brianLenghtInchesWhenHard = 6.75;
+enterButton.addEventListener('click', function(){
+  let softLenght = document.getElementById("softbox").value;
+  
```

Comment:
can we put a log right here on line 6? Just to see that this value is being accessed correctly?